app/models/stat.rb

```diff
@@ -0,0 +1,13 @@
+class Stat < ActiveRecord::Base
+  belongs_to :org
+
+  class << self
+    def to_csv(stats)
+      data = stats.reduce([]) do |acc, stat|
```

Comment:
Any reason why you didn't use `Array#map` here?

``` ruby
data = stats.map  { |acc, stat| { date: stat.date, count: stat.count } }
```

Reply:
just forgot, indeed I'm more inclined to create immutable data...